src/screens/HomeScreen.js

```diff
@@ -24,17 +25,17 @@ export default (props) => {
         <Button
           containerStyle={styles.button}
           titleStyle={styles.mainButtonText}
-          title="Jouer"
+          title="Play" //{i18n.t('buttons.play')}
```

Comment:
I'll introduce i18n, I already have it working locally. I just need to add it everywhere and push the code

Reply:
I opened #50, which will be ready during the week I think

When it is merged, there will be a single locales file containing all the app wording for a language (separated from the questions database)

Reply:
Hi Brandon!
Do you think you'll have time these days to introduce more English questions?

I think I'll finish the introduction of localization this September (there's only one issue with tests left)

Reply:
I've translated about 80 or so on a different branch

Reply:
Thank you very much for all these changes

I've been quite busy since September, I hope I find the time to add all the english questions to the app and release a new version within a few days/weeks